readme.md

```diff
@@ -125,8 +125,14 @@ TypeScript and Babel configs conflict each other, so you can only use one of the
 		exclude: /node_modules/
 	},
 	```
+3. Ask webpack to [resolve ts modules/imports](https://stackoverflow.com/questions/43595555/webpack-cant-resolve-typescript-modules) by adding this to your `webpack.config.json`:
+	```js
+	resolve: {
+	    extensions: ['.ts', '.js', '.json']
+	}
+	```
 
-1. Use the following as `tsconfig.json`, uses [sindresorhus/tsconfig][link-tsconfig] (install it as dependecy before using).
+4. Use the following as `tsconfig.json`, uses [sindresorhus/tsconfig][link-tsconfig] (install it as dependecy before using).
```

Comment:
Unrelated change, I do prefer having all numbers spelled out (in plain text files), but using version control and changes like this adding entries in between requires all other list items to be modified as well.

Change this and the other entries above back to `1.`

Reply:
✅